### PR TITLE
Jakarta Concurrency 3.0 Plan Review

### PR DESCRIPTION
**Pull request by smillidge**
Signed-off-by: Steve Millidge <steve.millidge@payara.fish>
## Plan Review PR template
When creating a specification project plan review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version.
- [x] An index page wombat/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
- [x] Title should be on the form "Jakarta Wombat X.Y (under development)"
- [x] Includes a plan for the release. Usually, a couple of statements is enough. Otherwise, a link to a document describing the release.
- [x] **N/A** Includes a detailed description of any backward incompatibility (Mark with **N/A** and check if none)
- [x] **N/A**Declares optional features if there are any (Mark with **N/A** and check if none)
- [x] Includes minimum Java SE version
- [x] **N/A** Describe any updates or required changes including splitting out the TCK (Mark with **N/A** and check if not planned)
- [x] Link to updated release record

### CODE REVIEW

**Comment by smillidge** [comment]
I would say at this stage I do not know. I will raise on the list whether we want the freedom to make breaking changes.

**Comment by edbratt** [comment]
The project might be able to make the case that the proposal to add Concurrency to Web Profile could be cause for declaring a major release. But, let's see what how the dev. community responds. Alternately the project might decide a dot release is good enough to start -- If things change, the project can run another ballot and update the proposal.

**Review by m0mus — APPROVED** [review]
LGTM

**Comment by smillidge** [comment]
We'd like to stick with the major at this point. However we may change through the development cycle given the type timescales. I assume this is possible in the process.

**Comment by kwsutter** [comment]
> We'd like to stick with the major at this point. However we may change through the development cycle given the type timescales. I assume this is possible in the process.

Sounds like you've given this decision due diligence, @smillidge.  If you decide during the development cycle that a 2.1 release is more appropriate than a 3.0 release, that will be relatively easy to accommodate.  Thanks!

**Review by edbratt — APPROVED** [review]
LGTM -- I'll start the ballot Friday (May 14) probably about noon PDT -- just any case there are any last minute changes.

File: concurrency/3.0/_index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+Bug Fixes as they arise during the release cycle
+
+Documentation clean up as identified during the release cycle
+
+Any updates required to meet the Java version requirements of Jakarta EE 10
+
+Any requirements identified by other specifications or the Jakarta EE 10 platform projects during the release cycle.
+
+* [Jakarta Concurrency 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.cu/releases/3.0.0)
```
Comment (kwsutter) [inline concurrency/3.0/_index.md:23]: Specifications should only have a two digit major.minor version.  Your text for the link is good (3.0), but the actual release record should also be 3.0 (not 3.0.0).  Thanks!

**Review by kwsutter — COMMENTED** [review]
Just a nit to clean up.  Thanks!

**Review by edbratt — APPROVED** [review]
Ballot successful. Will correct release-record text/link when updating with the ballot results. Thank you!

**Pull request merged**